specifyweb/frontend/js_src/lib/components/Forms/DateRange.tsx

```diff
@@ -9,22 +9,22 @@ import { parseAnyDate } from '../../utils/relativeDate';
 import type { RA } from '../../utils/types';
 import { filterArray } from '../../utils/types';
 import { sortFunction } from '../../utils/utils';
-import type { SpQueryField, Tables } from '../DataModel/types';
+import type { SpecifyResource } from '../DataModel/legacyTypes';
 import { serializeResource } from '../DataModel/serializers';
 import type { SpecifyTable } from '../DataModel/specifyTable';
 import { genericTables } from '../DataModel/tables';
+import type { SpQueryField, Tables } from '../DataModel/types';
 import { createQuery } from '../QueryBuilder';
 import { queryFieldFilters } from '../QueryBuilder/FieldFilter';
 import { QueryFieldSpec } from '../QueryBuilder/fieldSpec';
 import { flippedSortTypes } from '../QueryBuilder/helpers';
-import { SpecifyResource } from '../DataModel/legacyTypes';
 import { runQuery } from '../QueryBuilder/ResultsWrapper';
 
 export function DateRange({
   table,
   filterQueryField,
 }: {
-  //   readonly table: SpecifyModel;
+  //   Readonly table: SpecifyModel;
```

Comment:
not a fault of this PR, but could you please remove these commended out lines?
thanks